oscduplicator/duplicator.py

```diff
@@ -0,0 +1,248 @@
+import json
+import queue
+import socket
+from dataclasses import asdict
+from threading import Thread
+
+from pythonosc import dispatcher, osc_server, udp_client
```

Comment:
```suggestion
from pythonosc.dispatcher import Dispatcher
from pythonosc.osc_server import ThreadingOSCUDPServer
from pythonosc.udp_client import SimpleUDPClient
```

としておくと、後のコードがシンプルに書けそう